app/views/admin/divisions/_form.html.slim

```diff
@@ -72,6 +78,19 @@
     = documentation_popover(@documentations_by_html,
       html_identifier: "#{form_identifier}_public", options: popover_options)
 
+  - if @division.public?
```

Comment:
Putting this in an ERB if block means that when creating a new division you won't be able to set the URL. You'll have to know to save the record and then edit it. Is that ok?